apps/ws/src/app.module.ts

```diff
@@ -1,22 +1,20 @@
-import { join } from 'path';
-import { Module } from '@nestjs/common';
-import { RavenInterceptor, RavenModule } from 'nest-raven';
-import { APP_INTERCEPTOR } from '@nestjs/core';
-import { ServeStaticModule } from '@nestjs/serve-static';
+import { join } from "path";
+import { Module } from "@nestjs/common";
+import { ServeStaticModule } from "@nestjs/serve-static";
 import {
   createNestLoggingModuleOptions,
   LoggerModule,
   ProfilingModule,
-  TracingModule,
-} from '@novu/application-generic';
+  TracingModule
+} from "@novu/application-generic";
 
-import { AppController } from './app.controller';
-import { AppService } from './app.service';
-import { SharedModule } from './shared/shared.module';
-import { HealthModule } from './health/health.module';
-import { SocketModule } from './socket/socket.module';
+import { AppController } from "./app.controller";
+import { AppService } from "./app.service";
+import { SharedModule } from "./shared/shared.module";
+import { HealthModule } from "./health/health.module";
+import { SocketModule } from "./socket/socket.module";
 
-import packageJson from '../package.json';
+import packageJson from "../package.json";
```

Comment:
small question, was the single quotation to double quotation intentional?

Reply:
it's this missmatch betweeen preetier to eslint, I'll revert

Reply:
its preetier?